File: app/src/main/java/com/chooloo/www/callmanager/ui/contacts/ContactsMvpPresenter.java
package com.chooloo.www.callmanager.ui.contacts;

import com.chooloo.www.callmanager.entity.Contact;
import com.chooloo.www.callmanager.ui.base.MvpPresenter;

public interface ContactsMvpPresenter<V extends ContactsMvpView> extends MvpPresenter<V> {
    void onContactItemClick(Contact contact);

    boolean onContactItemLongClick(Contact contact);
}

File: app/src/main/java/com/chooloo/www/callmanager/ui/contacts/ContactsMvpPresenter.kt
package com.chooloo.www.callmanager.ui.contacts

import com.chooloo.www.callmanager.entity.Contact
import com.chooloo.www.callmanager.ui.base.MvpPresenter
import com.karumi.dexter.listener.PermissionGrantedResponse

interface ContactsMvpPresenter<V : ContactsMvpView> : MvpPresenter<V> {
    fun onContactItemClick(contact: Contact)
    fun onContactItemLongClick(contact: Contact): Boolean
}